src/classes/EP_ManageEPTemplate_CTRL.cls

```diff
@@ -0,0 +1,629 @@
+                new SelectOption('360', '6:00 AM'),
+                new SelectOption('390', '6:30 AM'),
+                new SelectOption('420', '7:00 AM'),
+                new SelectOption('450', '7:30 AM'),
+                new SelectOption('480', '8:00 AM'),
+                new SelectOption('510', '8:30 AM'),
+                new SelectOption('540', '9:00 AM'),
+                new SelectOption('570', '9:30 AM'),
+                new SelectOption('600', '10:00 AM'),
+                new SelectOption('630', '10:30 AM'),
+                new SelectOption('660', '11:00 AM'),
+                new SelectOption('690', '11:30 AM'),
+                new SelectOption('720', '12:00 PM'),
+                new SelectOption('750', '12:30 PM'),
+                new SelectOption('780', '1:00 PM'),
+                new SelectOption('810', '1:30 PM'),
+                new SelectOption('840', '2:00 PM'),
+                new SelectOption('870', '2:30 PM'),
+                new SelectOption('900', '3:00 PM'),
+                new SelectOption('930', '3:30 PM'),
+                new SelectOption('960', '4:00 PM'),
+                new SelectOption('990', '4:30 PM'),
+                new SelectOption('1020', '5:00 PM'),
+                new SelectOption('1050', '5:30 PM'),
+                new SelectOption('1080', '6:00 PM'),
+                new SelectOption('1110', '6:30 PM'),
+                new SelectOption('1140', '7:00 PM'),
+                new SelectOption('1170', '7:30 PM'),
+                new SelectOption('1200', '8:00 PM'),
+                new SelectOption('1230', '8:30 PM'),
+                new SelectOption('1260', '9:00 PM'),
+                new SelectOption('1290', '9:30 PM'),
+                new SelectOption('1320', '10:00 PM'),
+                new SelectOption('1350', '10:30 PM'),
+                new SelectOption('1380', '11:00 PM'),
+                new SelectOption('1410', '11:30 PM')
+            };
+            return options;
+        }
+        set;
+    }
+
+    /** @description Picklist options for the Task Priority field. We're using Task values directly in case they have
+    * been updated from their defaults.*/
+    public List<SelectOption> taskPriorityOptions {
+        get {
+            return listSelectOptions('Task','Priority');
+        }
+    }
+
+    /** @description Picklist options for the Task Status field. We're using Task values directly in case they have
+    * been updated from their defaults.*/
+    public List<SelectOption> taskStatusOptions {
+        get {
+            return listSelectOptions('Task','Status');
+        }
+    }
+
+    /** @description Returns a list of SelectOptions for a given object and picklist field name.
+    * @param objectName The name of the object.
+    * @param fieldName The name of the field.
+    * @return List<SelectOption> The select options for this picklist field.
+    */
+    private List<SelectOption> listSelectOptions(String objectName, String fieldName) {
```

Comment:
I coulda sworn we had these in a UTIL class, but I can't seem to find them.  I would be good to move these to UTIL so we can share them, since they are so general.  i won't hold up the pull request for this, however.